Makefile

```diff
@@ -9,6 +9,12 @@ else
 	PLATFORM := $(shell uname -s)
 endif
 
+ifeq ($(PLATFORM),Linux)
+	# used by ci-test
+	# (github actions only supports docker containers on linux)
+	SNO_POSTGRES_URL='postgresql://docker:docker@postgis:5432/gis'
```

Comment:
can you add this down at `ci-test` then? And can make it a target-specific variable if it's only used there.

eg:
```
ifeq ($(PLATFORM),Linux)
ci-test: export SNO_POSTGRES_URL ?= postgresql://docker:docker@postgis:5432/gis
endif

ci-test: 
    ...
```

Reply:
wow, Makefile is a _weird_ language. Done in latest